controllers/comments-controller.js

```diff
@@ -27,4 +31,14 @@ function createComments(req, res, next) {
     });
 }
 
-module.exports = { getComments, createComments };
+function deleteComment(req, res, next) {
+  removeComment(req.params.comment_id)
+    .then(() => {
+      res.status(204).send();
+    })
+    .catch((err) => {
+      next(err);
+    });
```

Comment:
some more syntactic sugar, if you only want to invoke `next` in the catch block: 
`.catch(next);` - express will know what to do and passes the error back to app

Reply:
thanks for all the refactor tips! Made these changes :)